src/OrleansTestingHost/AppDomainSiloHost.cs

```diff
@@ -92,13 +97,20 @@ public void UpdateGeneratedAssemblies(GeneratedAssemblies collection)
         /// <summary>Starts the silo</summary>
         public void Start()
         {
-            silo.Start();
+            this.silo.StartAsync().Wait();
```

Comment:
use `.GetAwaiter().GetResult()` instead of `.Wait()` here and in the Shutdown method